### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by maximzasorin** [nygardk/react-share#263]
Hello, when I run demo like this:

```
npm install && npm run run-demos
```

And go to:

```
http://localhost:8080/
```

I'm getting this error:
```
createShareButton.jsx:202 Uncaught TypeError: _react2.default.forwardRef is not a function
    at createShareButton (createShareButton.jsx:202)
    at Object.<anonymous> (FacebookShareButton.js:18)
    at Object../src/FacebookShareButton.js (main.78852a96d1ef0728a975.bundle.js:46695)
    at __webpack_require__ (bootstrap:723)
    at fn (bootstrap:100)
    at Object../src/index.js (main.78852a96d1ef0728a975.bundle.js:50271)
    at __webpack_require__ (bootstrap:723)
    at fn (bootstrap:100)
    at Object.<anonymous> (Demo.jsx:4)
    at Object../demo/Demo.jsx (main.78852a96d1ef0728a975.bundle.js:1278)
```

**Comment by elenapan** [comment]
The `forwardRef` function was added in react [16.3.0](https://github.com/facebook/react/releases/tag/v16.3.0). You should update `react` and `react-dom` packages to at least 16.3.0.
For me, 16.10.2 worked.
```
npm install react@16.10.2 --save
npm install react-dom@16.10.2 --save
```

**Comment by maximzasorin** [comment]
@elenapan Thank you, after the update react, it works. But the demo doesn't work if I just clone the repository, install the dependencies and run the demo.  I think this should be fixed.

**Comment by nygardk** [comment]
Fixed in `v4.0.0`.
